src/io.jl

```diff
@@ -497,78 +431,55 @@ Currently, grid voltages and line properties are ignored. This function places a
 voltage values.
 """
 function to_pandapower(grid::Grid,)
-    const LOAD_VOLT = 110 # kV # This is a crude approximation
-    const GEN_VOLT = 380 # kV # This is a crude approximation
+    LOAD_VOLT = 110 # kV, this is a crude approximation
+    GEN_VOLT = 380 # kV, this is a crude approximation
```

Comment:
If PandaPower has moved from kW to MW, these constants must be updated accordingly.